components/generic/SocialShareModal.stories.js

```diff
@@ -0,0 +1,42 @@
+import { storiesOf } from '@storybook/vue';
+import Vuex from 'vuex';
+import SocialShareModal from './SocialShareModal.vue';
+import ShareButton from './ShareButton.vue';
+
+const i18n = {
+  locale: 'en',
+  messages: {
+    en: {
+      actions: {
+        close: 'close',
+        share: 'share'
+      }
+    }
+  }
+};
+
+const store = new Vuex.Store({
```

Comment:
This appears not to do anything as removing it has no impact.

Let's try and get an actual URL into the share links instead. Use https://www.example.org/

Reply:
Try removing it and then clicking the share button you will the see the error `Cannot read property 'getters' of undefined`

Reply:
the socialShare story uses the canonicalUrl as well. Now socialShare unit tests are failing because example.org is expected but 'undefined' is returned